### DISCUSSION TITLE
请问 从1.4.0 升级到1.8.2需要注意什么

### DISCUSSION THREAD

**Discussion by idler0081** [apolloconfig/apollo#3775] [Q&A] [answered]
请问 从1.4.0 升级到1.8.2需要注意什么
刚接手了一个老项目，使用的apollo 1.4.0，为了修复xstream漏洞问题准备升级，请问客户端是否有改动，数据库是否需要调整
感谢

**Comment by Anilople** [comment, marked as answer]
可以参考 https://github.com/ctripcorp/apollo/releases 每次release 下的 How to upgrade from xxx to yyy

apollo-client 应该是可以直接升级的，config, admin, portal这些需要执行sql

**Reply by idler0081** [reply]
感谢解答
客户端还是需要升级的，对吧

**Reply by Anilople** [reply]
是的，apollo-client尽量和config service端的版本对齐
